### PR TITLE
Annual reports UI fixes

### PR DESCRIPTION
**Pull request by hryhoriivoloshyn**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: src/components/PrivateLayout/PrivateLayout.tsx [hunk 1]
```diff
@@ -54,15 +58,19 @@ const PrivateLayout = ({ children }: any) => {
...
       const imagePromise = UserApi.getImage(user.imagePath);
       const menuAccessPromise = UserApi.getUserMenuAccess(user.id);
-      const annualReportAccessPromise = AnnualReportApi.getUserAnnualReportAccess(user.id);
-      const statisticsAccessPromise = StatisticsApi.getUserStatisticsAccess(user.id);
+      const annualReportAccessPromise = AnnualReportApi.getUserAnnualReportAccess(
+        user.id
+      );
+      const statisticsAccessPromise = StatisticsApi.getUserStatisticsAccess(
+        user.id
+      );
```
Comment (tvardero) [inline src/components/PrivateLayout/PrivateLayout.tsx:66]: тут щось змінилось?

File: src/components/PrivateLayout/PrivateLayout.tsx [hunk 2]
```diff
@@ -99,364 +109,244 @@ const PrivateLayout = ({ children }: any) => {
           </div>
 
           <Menu theme="dark" mode="inline" className={classes.leftMenu}>
-
-            {userAccesses?.decisions
-              ? (
-                <Menu.Item
-                  key="decisions"
-                  icon={<SolutionOutlined />}
+            {userAccesses?.decisions ? (
```
Comment (tvardero) [inline src/components/PrivateLayout/PrivateLayout.tsx:112]: Це ти сам, чи це форматтер?

**Review by tvardero — APPROVED** [review]

**Review by vsynieok — APPROVED** [review]

File: src/components/PrivateLayout/PrivateLayout.tsx (hunk 1 above)
Comment (hryhoriivoloshyn) [reply src/components/PrivateLayout/PrivateLayout.tsx]: Maybe too late, but thanks. Yes, formatter just went wild, I am going to revert this changes.

File: src/components/PrivateLayout/PrivateLayout.tsx (hunk 2 above)
Comment (hryhoriivoloshyn) [reply src/components/PrivateLayout/PrivateLayout.tsx]: Yes, formatter just went wild, I am going to revert this changes.

**Review by Serhii-Buniak — APPROVED** [review]

**Pull request merged**